src/client/ui/cursor/iframe-cursor.js

```diff
@@ -0,0 +1,37 @@
+import hammerhead from './../deps/hammerhead';
+import CURSOR_UI_MESSAGES from './messages';
+import { sendRequestToFrame } from '../../runner/utils/iframe';
+
+var browserUtils = hammerhead.utils.browser;
+
+//HACK: we can't get element under cursor in cross-domain iframe in IE without hide cursor.
+//Instead make 'getElement' method asynchronous, we set cursor to position by one pixel farther.
+const RECOGNITION_INCREMENT = browserUtils.isIE ? 1 : 0;
+
+export function move (x, y) {
```

Comment:
Since it's some entity and not a bunch of utility methods it's should be exported as object:
```js
export default {
    move(x, y) {
    }
    ...
}
```